extensions/amp-experiment/1.0/test/test-attribute-mutation.js

```diff
@@ -75,71 +76,105 @@ describes.realWin(
         });
 
-        it('should allow color mutations', () => {
+        it('NOT allowed style mutations', () => {
           const attributeMutation = getAttributeMutationDefaultStyle(
-            'color: #000000'
+            'random: abc'
           );
-          expect(attributeMutation.validate()).to.be.equal(true);
+          expect(attributeMutation.parseAndValidate()).to.be.equal(false);
         });
 
-        it('should not allow !important mutations', () => {
-          const attributeMutation = getAttributeMutationDefaultStyle(
-            'color: #000000 !important;'
+        it('style mutations format', () => {
+          let attributeMutation = getAttributeMutationDefaultStyle('');
+          expect(attributeMutation.parseAndValidate()).to.be.equal(true);
+
+          attributeMutation = getAttributeMutationDefaultStyle(
+            'background-color red'
+          );
+          expect(attributeMutation.parseAndValidate()).to.be.equal(false);
+
+          attributeMutation = getAttributeMutationDefaultStyle(
+            'background-color ::red'
+          );
+          expect(attributeMutation.parseAndValidate()).to.be.equal(false);
+
+          attributeMutation = getAttributeMutationDefaultStyle(
+            'background-color: '
           );
-          expect(attributeMutation.validate()).to.be.equal(false);
+          expect(attributeMutation.parseAndValidate()).to.be.equal(true);
+
+          attributeMutation = getAttributeMutationDefaultStyle(
+            'background-color: red; ;'
+          );
+          expect(attributeMutation.parseAndValidate()).to.be.equal(true);
+
+          attributeMutation = getAttributeMutationDefaultStyle(
+            '  background-color: red      ;background-color:green;   '
+          );
+          expect(attributeMutation.parseAndValidate()).to.be.equal(true);
         });
 
-        it('should not allow HTML Comment mutations', () => {
-          const attributeMutation = getAttributeMutationDefaultStyle(
-            '<!-- color: #000000;'
+        it('background-color mutation', () => {
+          let attributeMutation = getAttributeMutationDefaultStyle(
+            'background-color: #000000'
+          );
+          expect(attributeMutation.parseAndValidate()).to.be.equal(true);
+
+          attributeMutation = getAttributeMutationDefaultStyle(
+            'background-color: red'
           );
-          expect(attributeMutation.validate()).to.be.equal(false);
+          expect(attributeMutation.parseAndValidate()).to.be.equal(true);
+
+          attributeMutation = getAttributeMutationDefaultStyle(
+            'background-color: invalid'
+          );
+          expect(attributeMutation.parseAndValidate()).to.be.equal(true);
         });
 
-        it('should not allow unallowed style mutations', () => {
+        it('color mutations', () => {
           const attributeMutation = getAttributeMutationDefaultStyle(
-            'position: fixed;'
+            'color: #000000'
```

Comment:
you might also need a test for 3-letter hex color
`color: #000`

Reply:
done